addSubcmd.go

```diff
@@ -15,10 +15,12 @@ func addSubcmd(args []string) {
 	addCmd := flag.NewFlagSet("add", flag.ExitOnError)
 
 	addCmd.Usage = func() {
+		addCmd.SetOutput(flag.CommandLine.Output())
```

Comment:
```suggestion
```
not here. when a wrong argument is given and this command is here, we're missing the error
```
flag provided but not defined: -blah
```